Sts1CobcSw/Edu/ProgramStatusHistory.hpp

```diff
@@ -55,13 +51,14 @@ inline constexpr std::size_t serialSize<edu::ProgramStatusHistoryEntry> =
 
 namespace edu
 {
-inline constexpr auto nProgramStatusHistoryEntries =
-    value_of(framSections.template Get<"eduProgramStatusHistory">().size)
-    / serialSize<ProgramStatusHistoryEntry>;
+inline constexpr auto nCachedProgramStatusHistoryEntries = 10;
 
-extern RODOS::RingBuffer<ProgramStatusHistoryEntry, nProgramStatusHistoryEntries>
+extern RingArray<ProgramStatusHistoryEntry,
+                 framSections.template Get<"eduProgramStatusHistory">(),
+                 nCachedProgramStatusHistoryEntries>
     programStatusHistory;
 
+inline constexpr auto nProgramStatusHistoryEntries = programStatusHistory.FramCapacity();
```

Comment:
Remove this constant. It is never used and not much shorter than `programStatusHistory.FramCapacity()` itself.